CONTRIBUTING.md

```diff
@@ -31,6 +31,31 @@ use GitHub pull requests for this purpose. Consult [GitHub
 Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+### Contribution guidelines and standards
+
+Before sending your pull request for 
+[review](https://github.com/tensorflow/swift-apis/pulls), 
+make sure your changes are consistent with the guidelines and follow the
+TensorFlow coding style.
```

Comment:
The TensorFlow coding style doesn't apply here because it's only for Python and C++. It's fine not to mention coding style here since it's part of guidelines.